CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Added
+- support for including customer data in check cart response by exposing the getCustomer method in checkCart
```

Comment:
```suggestion
- support for including customer data in cart validation by exposing the getCustomer method in checkCart
```

Reply:
check_cart it a rather technical term for an API action that most merchants never heard of.

Reply:
done